src/lib/es2022.intl.d.ts

```diff
@@ -88,4 +88,17 @@ declare namespace Intl {
          */
         supportedLocalesOf(locales: BCP47LanguageTag | BCP47LanguageTag[], options?: Pick<SegmenterOptions, "localeMatcher">): BCP47LanguageTag[];
     };
+
+    type EnumerationKeys = "calendar" | "collation" | "currency" | "numberingSystem" | "timeZone" | "unit";

```

Comment:
I don't know if we should be defining a type alias for this - especially one as broadly named as `EnumerationKeys`.

Reply:
I could inline it or just make it a string if preferred, but the proposal that added it is called [Intl enumeration](https://github.com/tc39/proposal-intl-enumeration) so I thought the name was fitting. 😄

Reply:
Yeah, I think for now let's inline it if that works.